governance/policies/archiving.md

```diff
@@ -15,10 +15,10 @@
 
 ### Compiler cutoff threshold
 
-The current compiler cutoff threshold is `4.08`.
+The current (2025-02-01) compiler cutoff threshold is `4.08`.
```

Comment:
I'm not sure how this date helps. AFAIU, whatever the current threshold is should be stated here, and when it is updates, this should be updated. If someone wants to see when this was written, they can look at git blame.

Reply:
My worry is that it will lead people to think that this has become "out of date" since this date will probably only change very infrequently. E.g., in 2027, this may still be the current cutoff threshold, but someone is likely to read it and think, "huh, current as of 2025, huh? Well that is quite old, this must be out of date." In other words, I don't think this date answers a questions anyone will have ("current just means current") but I think it is likely to raise irrelevant questions.

That said, it's not a big deal, so if you are sure this is an improvement, I am not really  worried about. If we find it to cause confusion later, we can deal with it at that time.

Reply:
I agree with @hannesm mostly because I am lazy and would never do the git blame 😅 

We have to update the threshold every now and then anyway, at that point we can update also the date. And if we see that it gets stale, we can always drop it later